android/src/main/kotlin/com/homex/open_mail_app/OpenMailAppPlugin.kt

```diff
@@ -153,4 +248,13 @@ class OpenMailAppPlugin : FlutterPlugin, MethodCallHandler {
 
 data class App(
         @SerializedName("name") val name: String
+)
+
+data class EmailContent (
+
+        @SerializedName("to") val to: List<String>,
+        @SerializedName("cc") val cc: List<String>,
+        @SerializedName("bcc") val bcc: List<String>,
+        @SerializedName("subject") val subject: String,
+        @SerializedName("body") val body: String
 )
```

Comment:
nit: According to ktlint,  File must end with a newline (\n)
```suggestion
)

```